applications/nrf_desktop/doc/dfu.rst

```diff
@@ -103,8 +103,8 @@ The DFU module implementation is centered around the transmission and the storag
 * `Partition preparation`_ - How the module prepares for receiving an image.
 
 The firmware transfer operation can also be carried out either by the :ref:`nrf_desktop_ble_smp` or :ref:`nrf_desktop_dfu_mcumgr`.
-The application user must not perform more than one firmware upgrade at a time.
-The modification of the data by multiple application modules would result in a broken image that would be rejected by the bootloader.
+Make sure that all enabled modules used for the firmware transfer operation use :ref:`nrf_desktop_dfu_lock` to synchronize non-volatile memory access.
+If a modules does not use the :ref:`nrf_desktop_dfu_lock`, the simultaneous modification of the data by the module and another application module would result in a broken image that would be rejected by the bootloader.
```

Comment:
```suggestion
If a module does not use the :ref:`nrf_desktop_dfu_lock`, simultaneous modification of the data by the module and another application module results in a broken image that would be rejected by the bootloader.
```

Reply:
Changed to `may result` to make information more accurate